Documentation/tools.md

```diff
@@ -7,5 +7,8 @@ weight: 70
 
 Rook provides a number of tools to help you manage your cluster.
 - [Toolbox](toolbox.md): A pod from which you can run all of the tools to troubleshoot the storage cluster
+- [Direct Tools](direct-tools.md): 
```

Comment:
the direct tools link has no description. add a short one because it looks odd without it: https://github.com/travisn/rook/blob/70f5fcac8a5acc978ca611df500b931199b517fe/Documentation/tools.md

Reply:
must've been friday afternoon